src/vs/code/electron-main/window.ts

```diff
@@ -29,6 +29,7 @@ export interface IWindowCreationOptions {
 	state: IWindowState;
 	extensionDevelopmentPath?: string;
 	allowFullscreen?: boolean;
+	macOSUseInlineToolbar: boolean;
```

Comment:
Yeah, that makes sense to me - yeah!

Any thoughts on what the name should be then? `toolbarStyle`?